avaframe/com4FlowPy/com4FlowPy.py

```diff
@@ -310,6 +312,9 @@ def mergeAndWriteResults(modelPaths, modelOptions):
     if modelOptions["infraBool"]:
         backcalc = SPAM.MergeRaster(modelPaths["tempDir"], "res_backcalc")
 
+    if modelOptions["forestInteractionBool"]:
```

Comment:
I suggest to drop the bool, see issue https://github.com/avaframe/AvaFrame/issues/330